From 4bfaf30f99a8f9f627ae88c7ceabed21b930875f Mon Sep 17 00:00:00 2001
From: Zmyeir <zr@zrlab.org>
Date: Tue, 20 Feb 2024 18:50:03 +0800
Subject: [PATCH] hwdb: Add support for MetawillBook01 to 60-sensor.hwdb Add
 accel orientation quirk for the METAPHYUNI MetawillBook01 2-in-1 laptop

(cherry picked from commit 8a461c572b792f6e40a67f0815466d844fb90d7e)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index b002be4ece..bbdb217440 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -773,6 +773,15 @@ sensor:modalias:acpi:KIOX020A*:dmi:*:rvnMEDION:rnMEDION*:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, -1
  ACCEL_LOCATION=base
 
+#########################################
+# METAPHYUNI
+#########################################
+
+# MetawillBook01
+sensor:modalias:acpi:KIOX000A*:dmi:*:rvnMETAPHYUNI:rnMetawillBook01:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, -1, 0; 0, 0, -1
+ ACCEL_LOCATION=display
+
 #########################################
 # MPMAN
 #########################################
